content/en/tracing/setup_overview/open_standards/otlp_ingest_in_the_agent.md

```diff
@@ -0,0 +1,233 @@
+
+
+{{< tabs >}}
+{{% tab "Host" %}}
+
+OTLP ingestion is off by default, and you can turn it on by updating your `datadog.yaml` file configuration or by setting environment variables. The following `datadog.yaml` configurations enable endpoints on the default ports. 
+
+For gRPC, default port 4317:
+
+```yaml
+otlp_config:
+  receiver:
+    protocols:
+      grpc:
+```
+For HTTP, default port 4318:
+
+```yaml
+otlp_config:
+  receiver:
+    protocols:
+      http:
+```
+
+Alternatively, configure the endpoints by providing the port through the environment variables:
+
+- For gRPC (`0.0.0.0:4317`): `DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT` 
+- For HTTP (`0.0.0.0:4318`): `DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT`
+
+These must be passed to both the core Agent and trace Agent if they are running in separate containers.
+
+Configure either gRPC or HTTP for this feature. Here is [an example application that shows configuration for both][1].
+
+<div class="alert alert-warning"><strong>Deprecation Notice</strong>: The use of `experimental:` for configuring OTLP receiving in `datadog.yaml` and the corresponding environment variables `DD_OTLP_HTTP_PORT` and `DD_OTLP_GRPC_PORT` is now deprecated and will be removed in Agent version `7.37.0`.</div>
+
+
+
+[1]: https://gist.github.com/gbbr/4a54dd02d34ad05e694952e0a02e1c67
+{{% /tab %}}
+{{% tab "Docker" %}}
+
+1. Follow the [Datadog Docker Agent setup][1]. 
+  
+2. For the Datadog Agent container, set the following endpoint environment variables and expose the corresponding port: 
+   - For gPRC: `DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT` and port `4317`
+   - For HTTP: `DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT` and port `4318`
+
+3. For the application container, set `OTEL_EXPORTER_OTLP_ENDPOINT` environment variable to point to the Datadog Agent container. For example:
+
+   ```
+   OTEL_EXPORTER_OTLP_ENDPOINT=http://<datadog-agent>:4318.
+   ```
+
+4. Both containers must be defined in the same bridge network, which is handled automatically if you use Docker Compose. Otherwise, follow the Docker example in [Tracing Docker Applications][2] to set up a bridge network with the correct ports.
+
+
+[1]: /agent/docker/
+[2]: /agent/docker/apm/#docker-network
+{{% /tab %}}
+{{% tab "Kubernetes (Daemonset)" %}}
+
+1. Follow the [Kubernetes Agent setup][1].
+
+2. Configure the following environment variables in both the trace Agent container and the core Agent container:
+   
+   For gPRC:
+   ```
+   name: DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT # enables gRPC receiver on port 4317
+   value: "0.0.0.0:4317"
+   ```
+
+   For HTTP:
+   ```
+   name: DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT # enables HTTP receiver on port 55681
+   value: "0.0.0.0:55681"
```

Comment:
Any reason we are using `55681` here instead of `4318`?

Reply:
I don't think so. I had forgot to update this in the internal doc. It's now been updated in the internal doc.

Reply:
Yep, we can switch to the standard port.